src/Blazored.LocalStorage/IStorageProvider.cs

```diff
@@ -20,4 +20,4 @@ internal interface IStorageProvider
         void SetItem(string key, string data);
         ValueTask SetItemAsync(string key, string data, CancellationToken? cancellationToken = null);
     }
-}
\ No newline at end of file
+}
```

Comment:
Can you revert this as it's just a whitespace change.

Reply:
Oops! Good catch.

Reply:
Reverted.